example_js_regex/lib/main.dart

```diff
@@ -20,23 +20,27 @@ class MyHomePage extends StatefulWidget {
 
 class _MyHomePageState extends State<MyHomePage> {
   void _runNlpExample() {
-    // const phrase = "It happened when the baby was only ten months old";
-    // const phrase = "I'll leave for 3h";
-    const phrase = "The project estimative is a 2 ys duration";
-    // const phrase = "We've been in Pakistan for 2ys";
-    // const phrase = "I'll leave for 1 year 1 month 21 days";
-    // const phrase = "I'll leave for 2 days 1 month";
+    const phrase = "We had a meeting 1 month and 21 days ago";
```

Comment:
I think I had all these commented out phrases so that I could quickly try them. Was there a reason to delete them?

Reply:
I shouldn't have deleted that. Added it back.